docs/README.md

```diff
@@ -1,123 +1,33 @@
-
-# Development
-
-## Components
-
-**[Datadog Agent](https://github.com/DataDog/datadog-agent)**: A service that runs on your application servers, accepting trace data from the Datadog Tracer and sending it to Datadog. The Agent is not part of this repo; it's the same Agent to which all Datadog tracers (e.g. Go, Python, Java, Ruby) send data.
-
-**[Datadog .NET Tracer](https://github.com/DataDog/dd-trace-dotnet)**: This repository. A set of .NET libraries that let you trace any piece of your .NET code. Supports manual instrumentation and can automatically instrument supported libraries out-of-the-box.
-
-## Windows
-
-### Minimum requirements
-
-- [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) or newer
-  - Workloads
-    - Desktop development with C++
-    - .NET desktop development
-    - .NET Core cross-platform development
-    - Optional: ASP.NET and web development (to build samples)
-  - Individual components
-    - .NET Framework 4.7 targeting pack
-- [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0)
-- Optional: [.NET Core 2.1 Runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1) to test in .NET Core 2.1 locally.
-- Optional: [.NET Core 2.2 Runtime](https://dotnet.microsoft.com/download/dotnet-core/2.2) to test in .NET Core 2.2 locally.
-- Optional: [nuget.exe CLI](https://www.nuget.org/downloads) v3.4.4 or newer
-- Optional: [WiX Toolset 3.11.1](http://wixtoolset.org/releases/) or newer to build Windows installer (msi)
-  - Requires .NET Framework 3.5 SP2 (install from Windows Features control panel: `OptionalFeatures.exe`)
-  - [WiX Toolset Visual Studio Extension](https://wixtoolset.org/releases/) to build installer from Visual Studio
-- Optional: [Docker for Windows](https://docs.docker.com/docker-for-windows/) to build Linux binaries and run integration tests on Linux containers. See [section on Docker Compose](#building-and-running-tests-with-docker-compose).
-  - Requires Windows 10 (1607 Anniversary Update, Build 14393 or newer)
-
-Microsoft provides [evaluation developer VMs](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines) with Windows 10 and Visual Studio pre-installed.
-
-### Building from a command line
-
-From a _Developer Command Prompt for VS 2019_:
+Windows and Linux Installers|[See releases](https://github.com/LightStep/ls-trace-dotnet/releases)
+`LightStep.Trace`|[![Datadog.Trace](https://img.shields.io/nuget/vpre/LightStep.Trace.svg)](https://www.nuget.org/packages/LightStep.Trace)
+`LightStep.Trace.ClrProfiler.Managed`|[![LightStep.Trace.ClrProfiler.Managed](https://img.shields.io/nuget/vpre/LightStep.Trace.ClrProfiler.Managed.svg)](https://www.nuget.org/packages/LightStep.Trace.ClrProfiler.Managed)
```

Comment:
Is `Lighstep.Trace.OpenTracing` missing here?

Reply:
It's not actually used, I'm not entirely sure why they have that project in the repo.

Reply:
Ah k.